keyboards/zinc/keymaps/toshi0383/keymap.c

```diff
@@ -0,0 +1,163 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
+    _LOWER,
+    _RAISE,
+    _ADJUST,
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
```

Comment:
You don't need this custom keycode, as you only have the QWERTY base layer (no Colemak/Dvorak/etc.).

```suggestion
  LOWER = SAFE_RANGE,
```